src/app/home/avatar-generator/avatar-generator.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component, OnInit, ViewChild, ElementRef, ChangeDetectionStrategy } from '@angular/core';
+import htmlToImage from 'html-to-image';
+import {avatarOptions, buildAvatar} from './avatar-utils';
+
+@Component({
+  selector: 'wgj-avatar-generator',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  templateUrl: './avatar-generator.component.html',
+  styleUrls: ['./avatar-generator.component.scss'],
+})
+export class AvatarGeneratorComponent implements OnInit {
+  avatar = avatarOptions;
+  avatarKeys = Object.keys(avatarOptions);
+  constructor() { }
+  @ViewChild('screen') screen: ElementRef;
+  @ViewChild('canvas') canvas: ElementRef;
```

Comment:
It seems that this reference is not necessary to download the image anymore, we could remove it.